Make keystone optional for ssgc functions (#3355)
keystonejs/keystone: .changeset/famous-ties-type.md

---
'@keystonejs/server-side-graphql-client': minor
'@keystonejs/keystone': patch
---

> - `context` object created via `keystone.createContext` now has `gqlNames(listKey)` function. This allows extracting graphql query and mutation names via context object.

Reviewer: ```suggestion
- Added a function `gqlNames(listKey)` to the `context` object created by `keystone.createContext`  This allows extracting graphQL query and mutation names from the `context` object.
```

> - Made keystone optional when context is provided in any of the utility functions in `server-side-graphql-client` package.

Reviewer: ```suggestion
- Made the `keystone` argument optional when a `context` value is provided in any of the utility functions in `server-side-graphql-client` package.
```
